src/main/java/org/softc/armoryexpansion/common/integration/aelib/config/MaterialConfigOptions.java

```diff
@@ -0,0 +1,106 @@
+package org.softc.armoryexpansion.common.integration.aelib.config;
+
+import org.softc.armoryexpansion.common.integration.aelib.plugins.general.material.IBasicMaterial;
+
+public class MaterialConfigOptions extends ArmorMaterialConfigOptions{
+    private ToolConfigOptions toolOptions;
+    private RangedConfigOptions rangedOptions;
+
+    MaterialConfigOptions() {
+        super("DEFAULT", true, true, false);
+        this.toolOptions = new ToolConfigOptions(false);
+        this.rangedOptions = new RangedConfigOptions(false);
+    }
+
+    MaterialConfigOptions(IBasicMaterial iBasicMaterial) {
+        super(iBasicMaterial);
+
+        this.toolOptions = new ToolConfigOptions(iBasicMaterial.isToolMaterial());
+        this.rangedOptions = new RangedConfigOptions(iBasicMaterial.isRangedMaterial());
+    }
+
+    public boolean isToolEnabled(){
+        return this.toolOptions.enableTool;
+    }
+
+    public boolean isHeadEnabled(){
+        return this.toolOptions.enableHead;
+    }
+
+    public boolean isHandleEnabled(){
+        return this.toolOptions.enableHandle;
+    }
+
+    public boolean isExtraEnabled(){
+        return this.toolOptions.enableExtra;
+    }
+
+    boolean isRangedEnabled(){
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/manual/YaibaToKen/Armory-Expansion/pullRequest?prid=3896610)